### ISSUE TITLE
Add support to the module to recognize automatically the current user logged in (if any)

### ISSUE THREAD

**Issue by mateusdemboski** [ricardoalcantara/raven-csharp-light#9]

**Comment by ricardoalcantara** [comment]
Do you mean asp.net logged in users?

**Comment by mateusdemboski** [comment]
Yes! 
Sometimes this information its too useful and i think that we can implement something that provides this information, like the official library

**Comment by ricardoalcantara** [comment]
I totally agree with you. But I was thinking a while ago to create another assembly (dll) for asp.net core, with Exception Middleware, IHttpContextAccessor and (now you said) users logged in. I also realized that commit you made about environment had the ASPNETCORE_ENVIRONMENT, it also could be moved to this new lib. I have started a branch a while ago and I didn't finished it because I was doing another project too, but I was thinking about coming back because I will need those features too.
https://github.com/ricardoalcantara/raven-csharp-light/tree/feature/asp-net-core-middleware/src/SharpRavenLight/src/SharpRavenLight.AspNetCore

**Comment by mateusdemboski** [comment]
@ricardoalcantara You can merge the master into SharpRavenLight.AspNetCore branch?
I will analyze some implementations to do. =)

**Comment by ricardoalcantara** [comment]
@mateusdemboski The merge has been done. And I apologize because I haven't had time to publish this new features yet, I just have access to my windows machine at night. But soon it will be publish.

**Comment by mateusdemboski** [comment]
@ricardoalcantara I created a new branch called "feature/asp-net-core-logger". This is based on a Logger Extension. Pls, verify if this implementation is valid.

**Comment by ricardoalcantara** [comment]
IMHO, it's a valid implementation, but the ILogger class could be used outside asp.net core.

**Comment by mateusdemboski** [comment]
Maybe we can use the logger to both versions (4.5 and core) and add factories to get the user separated.
Only i need a short time to think how to do it... =P

i'm looking to this files and how to implement in the library:
https://github.com/getsentry/raven-csharp/blob/develop/src/app/SharpRaven/Data/SentryUser.cs
https://github.com/getsentry/raven-csharp/blob/develop/src/app/SharpRaven/Data/SentryUserFactory.cs

**Comment by ricardoalcantara** [comment]
Okay! And the ILogger implementation can be hosted inside the core lib.
